### PR TITLE
Completed bootcamp, 67% accuracy

### PR DESCRIPTION
**Pull request by enjineerMan**
Might have to do with pool_size=(2,2)?

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -12,6 +12,42 @@
 
 # Import whatever libraries/modules you need
```
Comment (rayjinghaolei) [inline main.py:13]: Try to break up your code into different functions, in this case, it can be data_preparation(), model_construction(),model_training() and etc. You can be creative about the names. The purpose is that when you are actually contributing to our repo, you can write some clear and easy-to-debug code.

File: main.py [hunk 2]
```diff
@@ -12,6 +12,42 @@
...
+
+x_train = tf.keras.utils.normalize(x_train, axis=1)
+x_test = tf.keras.utils.normalize(x_test, axis=1)
+
+model = tf.keras.models.Sequential() 
+
+# First convolution layer
+model.add(Conv2D(256, (3, 3), input_shape=x_train.shape[1:]))
+model.add(Activation('relu'))
+model.add(MaxPooling2D(pool_size=(2,2)))
```
Comment (rayjinghaolei) [inline main.py:33]: Something for you to think about or do research: (write your answer in the comment)
What's Max Pooling 2D?
What does Max Pooling 2D do?

**Review by rayjinghaolei — CHANGES_REQUESTED** [review]

File: main.py (hunk 2 above)
Comment (enjineerMan) [reply main.py]: Pooling is a method of downsampling the image by decreasing the number of pixels. For each 2x2 patch in the image, the max-pooling method selects the pixel with the highest value and replaces the other 3 pixels in the 2x2 patch with that value. 
Correct me if I'm wrong!

File: main.py [hunk 3]
```diff
@@ -12,6 +12,52 @@
...
+
+def model_training():
+    model.fit(x_train, y_train, epochs=20)
+
+def model_validation():
+    val_loss, val_acc = model.evaluate(x_test, y_test) #validate model
+    print(val_loss, val_acc)
+
+if __name__ == "__main__":
+    data_preparation()
```
Comment (Shrinjay) [inline main.py:60]: This is not necessarily the best way to use these functions, but it's nothing that bad.  It's how I expect someone not as familiar with fp to write functions. Ideally though, you want functions to return and not affect any variables outside the function, this isn't possible sometimes but it's good to avoid it.

So the best way to do this would be: 

``` 
data_preperation():
   ... do stuff
   return (x_train, y_train), (x_test, y_test)

def model_construction():
   ... do stuff
   return model

def train_model(model, x_train, y_train):
    .... train model
   # No return here, so this function has side effects

def model_validation(model):
     .... evaluate model
     return val_loss, val_acc

if __name__ == "__main__":
    (x_train, y_train), (x_test, y_test) = data_preparation()
    model = model_construction()
    model_training(model)
    val_loss, val_acc = model_validation(model)
```
Anyways that's just me being an fp nerd. Well done otherwise
